packages/manager/apps/pci-quota/src/pages/quota/IncreaseQuotaSupport.page.tsx

```diff
@@ -0,0 +1,254 @@
+import { z } from 'zod';
+
+import { Button, Textarea } from '@datatr-ux/uxlib';
+import { zodResolver } from '@hookform/resolvers/zod';
+
+import { usePciUrl } from '@ovh-ux/manager-pci-common';
+import {
+  Links,
+  LinkType,
+  Notifications,
+  Title,
+  useMe,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import { useTracking } from '@ovh-ux/manager-react-shell-client';
+import { OdsMessage } from '@ovhcloud/ods-components/react';
+import { Controller, useForm } from 'react-hook-form';
+import { Translation, useTranslation } from 'react-i18next';
+import { useHref, useNavigate, useParams } from 'react-router-dom';
+
+import { TCreateTicketResponse } from '@/api/data/ticket';
+import {
+  useGetIssueTypes,
+  useMutationCreateTicket,
+} from '@/api/hooks/useIssueTypes';
+import {
+  SUPPORT_ISSUE_TYPE_IDS,
+  SUPPORT_TICKET_ID_URL,
+  TRACK,
+} from '@/constants';
+
+const IncreaseQuotaForm = ({
+  fields,
+  onSubmit,
+  isPending,
+}: {
+  fields: { id: number; label: string; isRequired: boolean }[];
+  goBackHref: string;
+  onSubmit: (data: { [id: number]: string }) => void;
+  isPending: boolean;
+}) => {
+  const { t } = useTranslation('quotas/increase');
+  const object = {};
+  fields?.forEach((field) => {
+    object[field.id] = field.isRequired
+      ? z.string().min(20, 'Veuillez saisir un minimum de 20 caractères.')
```

Comment:
translation missing ?

Also I think you can interpolate the min value inside the text